src/translations/data.json

```
@@ -681,8 +681,8 @@
     "uk": "Використовувати форми словозміни при пошуку міжслав'янською мовою",
     "be": "Выкарыстоўваць формы словазмянення пры пошуку на міжславянскай мове",
     "pl": "Szukaj także odmienionych form międzysłowiańskich słów",
-    "cs": "Používat inflexní formy slov při hledání v mezislovanském jazyce",
-    "sk": "Používať inflexné formy slov pri hľadaní v mezislovanském jazyku",
+    "cs": "Používat ohebné formy slov při hledání v mezislovanském jazyce",
+    "sk": "Používať ohebné formy slov pri hľadaní v mezislovanském jazyku",
```

ohEbne is czech, if we want this word it should be "ohybné" 
```suggestion
    "sk": "Používať ohybné formy slov pri hľadaní v mezislovanském jazyku",
```
but I believe better would be "formy skloňovania" or simply just "skloňnovanie"
```suggestion
    "sk": "Používať formy skloňovania slov pri hľadaní v mezislovanském jazyku",
```
```suggestion
    "sk": "Používať skloňovanie slov pri hľadaní v mezislovanském jazyku",
```

---

I chose the third option